### PR TITLE
added a user mention to click me function

### PR DESCRIPTION
**Pull request by Shane22435**

### CODE REVIEW

**Comment by wizzdom** [comment]
Very nice!

**Review by DistroByte — DISMISSED** [review]
LGTM!

File: src/extensions/hello_world.py [hunk 1]
```diff
@@ -94,9 +94,11 @@ async def components(self, ctx: discord.SlashContext):
     @discord.component_callback("click_me")
     async def click_me(self, ctx: discord.ComponentContext):
         """A callback for the click me button"""
-        await ctx.send("You clicked me!")
+        user = ctx.author
+        await ctx.send(f"{user.metion}, you clicked me!")
```
Comment (JedHazaymeh) [inline src/extensions/hello_world.py:98]: Spelling mistake here: `metion` -> `mention`

**Review by JedHazaymeh — CHANGES_REQUESTED** [review]

File: src/extensions/hello_world.py (hunk 1 above)
Comment (JedHazaymeh) [reply src/extensions/hello_world.py]: You can say it's fixed by resolving the conversation 🤣

**Pull request closed without merging**